shuup/core/models/_service_base.py

```diff
@@ -454,8 +455,13 @@ def get_delivery_time(self, service, source):
         return None
 
 
-class TranslatableServiceBehaviorComponent(six.with_metaclass(
-        PolyTransModelBase,
-        ServiceBehaviorComponent, TranslatableShuupModel)):
+if version.get_docs_version() in ["1.8", "1.9"]:
+    META_CLASS = six.with_metaclass(PolyTransModelBase, ServiceBehaviorComponent, TranslatableShuupModel)
+else:
+    META_CLASS = six.with_metaclass(PolyTransModelBase, ServiceBehaviorComponent, PolymorphicTranslatableShuupModel)
```

Comment:
This is not a meta class you're changing here, but the base class, so the name of this variable is misleading.

I think you should rather do something like this:
```python
import django


_translatable_model = (
    PolymorphicTranslatableShuupModel if django.VERSION >= (1, 10)
    else TranslatableShuupModel)


 class TranslatableServiceBehaviorComponent(six.with_metaclass(
        PolyTransModelBase,
        ServiceBehaviorComponent, _translatable_model)):
```